lib/chef/knife/ec2_server_create.rb

```diff
@@ -111,10 +111,7 @@ class Ec2ServerCreate < Knife
         :short => "-T T=V[,T=V,...]",
         :long => "--tags Tag=Value[,Tag=Value...]",
         :description => "The tags for this server. [DEPRECATED] Use --aws-tag instead.",
-        :proc => Proc.new { |v|
-          Chef::Log.warn("[DEPRECATED] --tags option is deprecated. Use --aws-tag option instead.")
-          v
-        }
+        :proc => Proc.new { |tags| tags.split(',') }
```

Comment:
why we have to remove the deprecation warning from here? can't we do tags.split(',') along with deprecation warning here?